Once I import atomWithStorage I get 2 weird TS errors

When adding `import { atomWithStorage } from 'jotai/utils'` (even with keeping it unused) I get this...

```
../../node_modules/.pnpm/jotai@1.2.1_react@17.0.2/node_modules/jotai/utils/atomWithObservable.d.ts(13,5): error TS1170: A computed property name in a type literal must refer to an expression whose type is a literal type or a 'unique symbol' type.
../../node_modules/.pnpm/jotai@1.2.1_react@17.0.2/node_modules/jotai/utils/atomWithObservable.d.ts(13,13): error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.
```

--- a/src/utils/atomWithObservable.ts
+++ b/src/utils/atomWithObservable.ts
@@ -1,6 +1,12 @@
 import { atom } from 'jotai'
 import type { Atom, Getter, WritableAtom } from 'jotai'
 
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol
+  }
+}
+
 type Subscription = {
   unsubscribe: () => void
 }

